From f8bcea1e76ecacf658976158c65e05297df737f9 Mon Sep 17 00:00:00 2001
From: Chen Baozi <chenbaozi@phytium.com.cn>
Date: Mon, 17 Jun 2024 19:33:08 +0800
Subject: [PATCH 096/150] dt-bindings: net: dwmac: Add bindings for Phytium
 DWMAC

Add bindings for Phytium DWMAC controllers, which are based
on snps,dwmac.

Signed-off-by: Chen Baozi <chenbaozi@phytium.com.cn>
Change-Id: I83ededc6f0dd7acc07ba55da7d94436fbc7ed26c
Signed-off-by: Andrew Powers-Holmes <aholmes@omnom.net>
---
 Documentation/devicetree/bindings/net/snps,dwmac.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/net/snps,dwmac.yaml b/Documentation/devicetree/bindings/net/snps,dwmac.yaml
index 5c2769dc689a..f7c8b435a2ec 100644
--- a/Documentation/devicetree/bindings/net/snps,dwmac.yaml
+++ b/Documentation/devicetree/bindings/net/snps,dwmac.yaml
@@ -66,6 +66,7 @@ properties:
         - ingenic,x2000-mac
         - loongson,ls2k-dwmac
         - loongson,ls7a-dwmac
+        - phytium,gmac
         - qcom,qcs404-ethqos
         - qcom,sa8775p-ethqos
         - qcom,sc8280xp-ethqos
-- 
2.47.0

